README.md

```diff
@@ -323,15 +289,12 @@ lifting for you.
 ```php
 <?php
 
-class MyController extends Controller
+class MyController extends AbstractController
 {
-    public function indexAction()
-    {
-        /** @var FilterService */
-        $imagine = $this
-            ->container
-            ->get('liip_imagine.service.filter');
+use Liip\ImagineBundle\Service\FilterService;
```

Comment:
this should go above the class declaration

Reply:
yeah, that's what I get for writing PHP directly in the README!